examples/google-analytics/app/routes/contact.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { Form, useSubmit } from "remix";
+import type { ActionFunction } from "remix";
+
+import * as gtag from "~/utils/gtags.client";
+
+export const action: ActionFunction = () => {
+  return "";
+};
+
+export default function Contact() {
+  const submit = useSubmit();
+
+  const handleSubmit = (
+    e: React.SyntheticEvent<
+      | HTMLFormElement
+      | HTMLButtonElement
+      | HTMLInputElement
+      | FormData
+      | URLSearchParams
+      | {
+          [name: string]: string;
+        }
+      | null
+    >
```

Comment:
I'm pretty sure this submit will only be `HTMLFormElement` right?
```suggestion
    e: React.SyntheticEvent<HTMLFormElement>
```